geomstats/geometry/hypersphere.py

```diff
@@ -16,21 +16,7 @@
 from geomstats.geometry.riemannian_metric import RiemannianMetric
 
 TOLERANCE = 1e-6
-EPSILON = 1e-8
-
-COS_TAYLOR_COEFFS = [1., 0.,
-                     - 1.0 / math.factorial(2), 0.,
-                     + 1.0 / math.factorial(4), 0.,
-                     - 1.0 / math.factorial(6), 0.,
-                     + 1.0 / math.factorial(8), 0.]
-INV_SIN_TAYLOR_COEFFS = [0., 1. / 6.,
-                         0., 7. / 360.,
-                         0., 31. / 15120.,
-                         0., 127. / 604800.]
-INV_TAN_TAYLOR_COEFFS = [0., - 1. / 3.,
-                         0., - 1. / 45.,
-                         0., - 2. / 945.,
-                         0., -1. / 4725.]
+EPSILON = 1e-4
```

Comment:
Why this change in EPSILON?

Reply:
Because this is the precision of some functions in Pytorch, e.g. gs.cos(1e-4) == 1. results in True